src/main/java/com/kingland/eip/javaFundamental/FundamentalTest.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright 2020 Kingland Systems Corporation. All Rights Reserved.
+ */
+package com.kingland.eip.javaFundamental;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+/**
+ *  This main function can solve four issues
+ *  1.print hello world
+ *  2.print pattern and current date&time
+ *  3.print month abbreviation
+ *  4.print and check Good morning string
+ */
+public class FundamentalTest {
```

Comment:
PR1: Please think about using junit to write test code, and move test code to test folder.